t/unit/utils/test_time.py

```diff
@@ -253,3 +254,41 @@ def test_utcoffset(self, patching):
         assert utcoffset(time=_time) is not None
         _time.daylight = False
         assert utcoffset(time=_time) is not None
+
+
+class test_get_exponential_backoff_interval:
+
+    @patch('random.randrange', lambda n: n - 2)
```

Comment:
By intuition alone, it seemed more appropriate to not use edge case values (the endpoint) here, as I would like to display that jitter **does** affect the result.